content/docs/reference/design-docs/timetable.en.md

```diff
@@ -86,21 +127,44 @@ path:
  - {id: c, deleted: true, waypoint: true, trigram: ABC}
  - {id: d, waypoint: true, track: toto, offset: 42}
 
-engineering_margins:
- - {from: b, to: c, value: "2%", distribution: MARECO}
+stops:
+ - {at: a, for: PT5M}
 
-standard_margins:
-  # the begin and end waypoints are always implicitly added
+# there are two mutually exclusive ways to add time constraints:
+# margins and arrival times. Both methods share parameters, such
+# as the distribution, intermediate waypoints, and transitions.
```

Comment:
We should check that these assertion are correct:
- Having a single way of computing margins
- Having two mutually exclusive ways to add time constraints